zookeeper-jute/src/main/java/org/apache/jute/ToStringOutputArchive.java

```diff
@@ -32,6 +32,7 @@ public class ToStringOutputArchive implements OutputArchive {
 
     private PrintStream stream;
     private boolean isFirst = true;
+    private long dataSize;
```

Comment:
Need initialize to 0

Reply:
I don't understand this comment @hangc0276 
the default value of a long is always 0